shell/machine-config/harvester.vue

```diff
@@ -156,6 +150,14 @@ export default {
             });
           }
         });
+
+        try {
+          const { data: nodes } = await this.$store.dispatch('cluster/request', { url: `${ url }/${ NODE }s` });
```

Comment:
Why did this request need to move out of the `allHashSettled` above?

Reply:
I think we need to catch the errors of node's request since the response of node may have `404/500` error code.

Reply:
Sorry for I missed the link to the issue [2567](https://github.com/harvester/harvester/issues/2567) that the local cluster owner has no permission to access the node resource.